thumbor/handlers/__init__.py

```diff
@@ -39,6 +39,13 @@
 except NameError:
     basestring = str  # Python 3
 
+if sys.version_info.major >= 3:
```

Comment:
I think it would be nice to keep a version check standard. Here we have `sys.version_info.major`, and here https://github.com/thumbor/thumbor/blob/6292cb58b866b8267fba1c0a5974a0f05d7963ac/thumbor/engines/extensions/pil.py#L86 with `six`.

Reply:
Maybe we should just drop Py2 compatibility all together. Less than 2 months left for Py2 support anyway. 
Thoughts?

Reply:
I agree, I think it's worth removing support for python 2.